.github/workflows/test.yml

```diff
@@ -24,9 +24,6 @@ jobs:
         with:
           go-version: ${{matrix.go}}
 
-      - name: build
```

Comment:
Now we do not build pkg in workflow. Why?

Reply:
It wasn't built before as well make had mostly empty all target.